src/main/java/com/easypost/easyvcr/Utilities.java

```diff
@@ -39,73 +34,62 @@ public static boolean responseCameFromRecording(HttpURLConnection connection) {
 
     /**
      * Check if the object is a dictionary.
+     * DEPRECATED: Not intended for public use.
+     *
      * @param obj The object to check.
      * @return True if the object is a dictionary.
      */
+    @Deprecated
     public static boolean isDictionary(Object obj) {
-        return obj instanceof Map;
+        return com.easypost.easyvcr.internal.Utilities.isDictionary(obj);
     }
 
     /**
      * Check if the object is a list.
+     * DEPRECATED: Not intended for public use.
+     *
      * @param obj The object to check.
      * @return True if the object is a list.
      */
+    @Deprecated
     public static boolean isList(Object obj) {
-        return obj instanceof List;
+        return com.easypost.easyvcr.internal.Utilities.isList(obj);
     }
 
     /**
      * Remove elements from a JSON string.
+     * DEPRECATED: Not intended for public use.
+     *
      * @param json The JSON string to remove elements from.
      * @param elements The elements to remove.
      * @return The JSON string without the elements.
      */
+    @Deprecated
```

Comment:
## Author's Note

These functions should never have been public facing in the first place. But, sticking with the Java spirit of preserving backwards compatibility until the end of time, these functions are simply deprecated, but still work if an end-user does need them.

Reply:
This tool hasn't hit v1 yet so we are under no obligation to preserve expectations. Let's just flip them to private and avoid the deprecation bits altogether.